acceptance/tests/environment/custom_type_provider_from_same_environment.rb

```diff
@@ -2,6 +2,10 @@
 require 'puppet/acceptance/environment_utils'
 extend Puppet::Acceptance::EnvironmentUtils
 
+tag 'audit:medium',
+    'audit:acceptance',
```

Comment:
Yeah, we looked at this one together, but I agree with your new assessment.
It currently is using the `mk_temp_environment_with_teardown` helper.